Lib/collections/abc.py

```diff
@@ -1,3 +1,3 @@
-from _collections_abc import *
-from _collections_abc import __all__  # noqa: F401
-from _collections_abc import _CallableGenericAlias  # noqa: F401
```

Comment:
Do you know why we imported `_CallableGenericAlias`? was it for a legacy usage in typing?

Reply:
This import was added in 463c7d3d149283814d879a9bb8411af64e656c8e. I do not think that it was needed at first place. In any case, this PR does not change its accessibility.